core/tests/unit/test_exceptions.py

```diff
@@ -14,6 +14,8 @@
 
 import unittest
 
+from google.cloud.exceptions import _HAVE_GRPC
```

Comment:
If this is the only use of `_HAVE_GRPC`, define it in the test module rather than the module we export to users